docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,28 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+[Upcoming Feature] For the Soroban RPC, API endpoints will provide JSON format to unpack XDR base64 strings as long as a flag for `xdrFormat: "json"` is passed to the endpoints.
+
+## JSON and XDR Conversion Schema
+
+:::info
+
+SDF has published libraries which support the canonical and round-trippable conversion from XDR to JSON, and from JSON to XDR, without loss of information.
+
+:::
+
+The canonical JSON and XDR Schema is defined by the [rs-stella-xdr library](https://github.com/stellar/rs-stellar-xdr) and by the [@stellar/stellar-xdr-json-web](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web) npm package.
```

Comment:
nit, typo:

```suggestion
The canonical JSON and XDR Schema is defined by the [rs-stellar-xdr library](https://github.com/stellar/rs-stellar-xdr) and by the [@stellar/stellar-xdr-json-web](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web) npm package.
```

You can also call it the "`stellar_xdr` crate" and link to the [Rust docs](https://docs.rs/stellar-xdr/21.1.0/stellar_xdr/index.html) which might be a little more accessible.

Reply:
Good catch thanks